library/cwm/src/lib/cwm/table.rb

```diff
@@ -33,17 +60,19 @@ class Table < CustomWidget
     #   def items
     #     [
     #       [:first_user, "Joe", "Doe"],
-    #       [:best_user, "Chuck", "Norris"]
+    #       TableItem.new(:best_user, ["Chuck", "Norris"])
     #     ]
     #   end
```

Comment:
please add also example with nested items.

Reply:
Done. Both here and in the `examples` directory.